cli/cmd/tapRunner.go

```diff
@@ -337,7 +340,7 @@ func syncApiStatus(ctx context.Context, cancel context.CancelFunc, tappingOption
 		default:
 			err = controlSocket.SendNewTappedPodsListMessage(currentlyTappedPods)
 			if err != nil {
-				fmt.Printf("error Sending message via control socket %s\n", err)
+				fmt.Printf("error Sending message via control socket %v, error: %s\n", controlSocketStr, err)
```

Comment:
maybe this should be debug

Reply:
Just because it is CLI and it will be visible to the user (and I'm not sure we want to write the websocket url

Reply:
changed it to debug